index.bs

```diff
@@ -795,6 +795,128 @@ the following list:
+   defined by The WebSocket protocol.  [[!WSP]]
+
+  : {{WebSocketStreamOptions/signal}}
+  :: An {{AbortSignal}} that can be used to abort the handshake. After the handshake is complete the
+   signal does nothing.
+
+ : <code>|socket| . {{WebSocketStream/url}}</code>
+ :: Returns the <a lt="url">URL that was used</a> to establish the WebSocket connection.
+
+ : <code>|socket| . {{WebSocketStream/opened}}</code>
+ :: Returns a {{promise}} which resolves when the handshake successfully completes, or rejects if
+  the handshake fails. On success, it resolves to an object with the following properties:
+
+  : {{WebSocketOpenInfo/readable}}
+  :: A {{ReadableStream}} that can be used to read messages from the server. Each chunk read
+   corresponds to one message. Text messages will be read as strings; binary messages will be read
+   as ArrayBuffer objects.
+
+   The stream can be closed by calling {{ReadableStream/cancel()}} on
+   {{WebSocketOpenInfo/readable}}. If the argument passed to {{ReadableStream/cancel()}} is an
+   object with a {{WebSocketCloseInfo/code}} property and an <span class=allow-2119>optional</span> {{WebSocketCloseInfo/reason}}
+   property then {{WebSocketCloseInfo/code}} will be used as [=the WebSocket connection close
+   code=] and {{WebSocketCloseInfo/reason}} or the empty string will be used as [=the WebSocket
+   connection close reason=].
+
+   If no messages are read, or if messages are read slower than they are sent, then backpressure
+   will be applied and eventually the server will stop sending new messages.
+
+  : {{WebSocketOpenInfo/writable}}
+  :: A {{WritableStream}} that can be used to send messages to the server. Each chunk written will
+   be converted to one messages. Strings will be sent as text messages; {{BufferSource}} chunks will
+   be sent as binary messages.
+
+   The WebSocket can be closed by calling {{WritableStream/close()}} on
+   {{WebSocketOpenInfo/writable}}.
+
+   The stream can also be closed by calling {{WritableStream/abort()}}
+   {{WebSocketOpenInfo/writable}}. If an argument is passed to {{WritableStream/abort()}} then it
+   can be used to specify [=the WebSocket connection close code=] and [=the WebSocket connection
+   close reason=] as with {{ReadableStream/cancel()}} above.
+
+  : {{WebSocketOpenInfo/extensions}}
+  :: The [=extensions in use=] for the connection.
+
+  : {{WebSocketOpenInfo/protocol}}
+  :: The [=subprotocol in use=] for the connection.
+
+ : <code>|socket| . {{WebSocketStream/closed}}</code>
+ :: A {{promise}} which resolves when the connection is closed. If the connection did not close
+  [=cleanly=] then the promise is rejected. When the connection closes [=cleanly=] the promise is
+  resolved with an object with properties {{WebSocketCloseInfo/code}} and
+  {{WebSocketCloseInfo/reason}}, giving [=the WebSocket connection close code=] and [=the WebSocket
+  connection close reason=] the were supplied by the server.
```

Comment:
nit: Maybe remove "the" from "the were supplied by the server"? or replace "the" by "that"?

Reply:
Replaced with "that", thank you!